app/controllers/v0/sessions_controller.rb

```diff
@@ -119,7 +119,7 @@ def saml_login_relay_url
 
     def after_login_actions
       set_sso_cookie!
-      async_create_evss_account
+      AfterLoginJob.perform_async(@current_user&.uuid)
       create_user_account
```

Comment:
shouldn't `create_user_account` also live in here?

Reply:
`create_user_account` is for another team's code and i don't what depends on it/if it's ok to make it async

Reply:
@hpjaj I believe you worked on this... here is some background:

What we're trying to do is create a single background job / worker called "AfterLoginJob," in the future this job should be easily extendable to support N number of arguments.

In its current form it receives one argument, I think we need to change that to receive a hash instead. The keys of that hash will contain the keys needed for each individual job it will run. The idea here is that the background worker runs other background workers asynchronously. We don't want anything synchronous to be liable for preventing a user to sign-in. 

@lihanli , would you mind if @hpjaj pulls your branch and contributes changes directly?

Reply:
it's also important that the future code pushes are reverse compatible. In other words, If I push code that expands the worker to support additional hashes, but there are some already in memory that don't have those hash keys yet available, then it should fail gracefully.

Reply:
i don't mind, i can also just move `create_user_account` to `AfterLoginJob` if he thinks it's ok

Reply:
Moving `create_user_account` into the background job should be fine.  The only minor piece you'll want to be aware of is that the concern it resides in assumes `@current_user`:

https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/controllers/concerns/accountable.rb#L13

Outside of that, `create_user_account` is:

1. Checking for a user's `Account` record in Redis
2. If it is not present, it will run a `Account.find_or_create_by!` with the `@current_user`, and then cache it